<launch>
    <node
        name="coordinate_transform"
        pkg="ground_truth"
        type="coordinate_transform.py"
        output="screen">
        <param name="config_file" value="coordinate_transform_config.yaml" />
        <param name="environment" value="sim" />
    </node>
    <include file="$(find ground_truth)/launch/sphinx_ros_bridge.launch"></include>
</launch>